packages/paste-website/src/components/component-status/PackageStatus.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+
+interface PackageStatusProps {
+  status: string;
+}
+
+const PackageStatus: React.FC<PackageStatusProps> = ({status}) => {
```

Comment:
Figma status and package status are the same right? implementation wise. Any reason to have 2 components?

Reply:
I've combined these into one AssetStatus component.